pkg/deploy/generator/scripts/gatewayVMSS.sh

```diff
@@ -1,12 +1,20 @@
+# We need to manually set PasswordAuthentication to true in order for the VMSS Access JIT to work
+echo "setting ssh password authentication"
+sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config
+systemctl restart sshd.service
```

Comment:
Do we need to restart and could we get away with `reload` instead?

Reply:
We can move to `reload` if it's preferred, for our case either should work (I just had more confidence in the restart)